depclean-core/src/main/java/se/kth/depclean/core/analysis/asm/ResultCollector.java

```diff
@@ -28,7 +28,7 @@
  */
 public class ResultCollector {
 
-  private final Set<String> classes = new HashSet<>();
+  public final Set<String> classes = new HashSet<>();
```

Comment:
Why is it public?

Reply:
Actually, when I was writing this test, I used this set directly into it, so I make it public, but then I saw that there is a getter for that, and then I used this getter but forgot to fix the access modifier